docs/changelog/1623.md

```diff
@@ -0,0 +1 @@
+- Add new API method getMaxTimeStepSize() and to simplify return values and usability of advance(double dt) and initialize().
```

Comment:
It will be in the past :wink: 

```suggestion
- Added new API method getMaxTimeStepSize() and to simplify return values and usability of advance(double dt) and initialize().
```